.github/workflows/go.yml

```
@@ -183,13 +192,13 @@ jobs:
         run: ci/scripts/go_build.sh $(pwd)
       - name: Test
         shell: bash
-        run: ci/scripts/go_test.sh $(pwd)
+        run: ci/scripts/go_test.sh $(pwd)      
```

```suggestion
        run: ci/scripts/go_test.sh $(pwd)
```
Maybe try saving with "strip trailing whitespace"? Just trying to prevent future whitespace diffs